sycl/test-e2e/LLVMIntrinsicLowering/bitreverse.cpp

```diff
@@ -2,6 +2,10 @@
 
 // UNSUPPORTED: hip || cuda
 
+// TODO: Remove XFAIL after fixing
+// https://github.com/intel/intel-graphics-compiler/issues/330
+// XFAIL: gpu-intel-gen12
```

Comment:
Is this test passing on other GPUs (like pvc, dg2)?

Reply:
On commit 1ad567a3f7f47920549bb58d6b433b7af869c7dd, it was only failing on:

Windows: [e2e / Intel GEN12 Graphics with Level Zero]
Linux: [test (Intel, ["Linux", "gen12"]

Everything is passing now.

Reply:
We test only on Gen12 in pre-commit testing, so because you marked the test with XFAIL: gpu-intel-gen12, it passes now on Gen12. Unfortunately, it doesn't mean that test passes on other hardware. So it may easily fail on DG2 testing in post-commit.

Reply:
Also, does it make sense to land this PR since IGC has some issues with these instructions?
Basically, because of this you disable testing of all other builtins (for other types).

Maybe it worth outlining this builtings to a separate test? And maybe even check only llvm-spirv translation for now (not e2e)?

Reply:
Test has a segmentation fault on pvc machine and dg2 machine.

Reply:
2/4-bit bitreverse testing moved to new file.